src/components/modals/Debug.js

```diff
@@ -92,7 +92,7 @@ class Debug extends Component<*, *> {
           <ModalBody onClose={onClose}>
             <ModalTitle>developer internal tools</ModalTitle>
             <ModalContent>
-              <Box style={{ height: 60, overflow: 'auto' }}>
+              <Box>
```

Comment:
i would prefer we keep the scrolling thing because it allow to see the logs at same time